leshan-core/src/main/java/org/eclipse/leshan/core/request/ObserveCompositeRequest.java

```diff
@@ -136,4 +136,46 @@ public ContentFormat getResponseContentFormat() {
     public Map<String, String> getContext() {
         return context;
     }
+
+    @Override
+    public final String toString() {
+        return String.format(
+                "ObserveCompositeRequest [paths=%s requestContentFormat=%s responseContentFormat=%s, context=%s]",
```

Comment:
I see there is a `,` just after `responseContentFormat=%s` but not after the others.

So I checked in code base and there is case where `,` is used and so other where it is not used.
But when you generate `toString` with eclipse, `,` is used.

So I will add it